internal/app/components/roles/snew.go

```diff
@@ -2,15 +2,20 @@ package roles
 
 import "github.com/gzcharleszhang/course-planner/internal/app/components/permissions"
 
+const SnewDisplayName = "super_admin"
+const SnewId = "super_admin"
+
 // regular user
```

Comment:
I'm not sure what this comment is supposed to mean after seeing it on all the three roles, but I'm assuming it was just forgotten. Might be good to change this to a more descriptive comment on what the Snew role has permissions to do.
Otherwise, if it's actually semantically correct, it's a little confusing and might warrant a change.

Reply:
it was a copy pasta